tekton/ci/jobs/tekton-github-tasks-completed.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-tasks-completed
+  namespace: tektonci
+  description: |
+    Verifies that a PR has all the tasks completed
+spec:
+  params:
+    - name: body
+      description: The body of the Pull Request
+  stepTemplate:
+    env:
+      - name: HOME
+        value: /tekton/home
+  steps:
+    - name: install-pyyaml
+      image: python:3-alpine
+      script: |
+        pip install pyyaml --user
+    - name: check-github-tasks-completed
+      image: python:3-alpine
+      script: |
+        #!/usr/bin/env python
+        import sys
+        prBodyText = """$(params.body)"""
+        incompleteTasks = prBodyText.count("- []")
```

Comment:
Suggest counting lines that start with these characters (or whitespace + these characters) rather than "any" instance of them because I think they might show up naturally in some kinds of pull request descriptions. Here's a snippet of a PR body I'm imagining that could be caught by this accidentally:

```
I changed the type of the function arguments - []string to []int - in order to...
```

I _think_ (though could be wrong!) that the `- []string` bit would be counted in `incompleteTasks`?

Reply:
great idea - updated the check!